File: scripts/t3_hash_passwords.js
const bcrypt = require('bcrypt');
const saltRounds = 12;

async function hashPasswords() {
    const tester1Password = await bcrypt.hash('Pc4RM0AMKy5aSGfD', saltRounds);
    const tester2Password = await bcrypt.hash('4LWs6xnc1t32BzXA', saltRounds);
    console.log({ tester1Password, tester2Password });
}

hashPasswords();

File: scripts/t3_hash_passwords.py
import bcrypt

def hash_passwords():
    salt_rounds = 12

    # Plain text passwords
    tester1_password = "Pc4RM0AMKy5aSGfD"
    tester2_password = "4LWs6xnc1t32BzXA"

    # Hash passwords
    tester1_hashed = bcrypt.hashpw(tester1_password.encode('utf-8'), bcrypt.gensalt(salt_rounds)).decode('utf-8')
    tester2_hashed = bcrypt.hashpw(tester2_password.encode('utf-8'), bcrypt.gensalt(salt_rounds)).decode('utf-8')

    # Print hashed passwords
    print({"tester1Password": tester1_hashed, "tester2Password": tester2_hashed})

if __name__ == "__main__":
    hash_passwords()
